.github/workflows/slp-common-acceptance-test.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Acceptance action for lossprevention services'
+on: 
+  workflow_call:
+    inputs: 
+      secret-auth:
+        description: 'SECRET_AUTH value from secrets'
+        type: string
+        required: true
+      gcloud-auth-staging:
+        description: 'GCLOUD_AUTH_STAGING value from secrets'
+        type: string
+        required: true
+      collection:
+        description: 'Collection to use'
+        type: string
+        required: true
+        default: './slp/tests/acceptance/integration_test.postman_collection.json'
+      environment:
+        description: 'Environment to use'
+        type: string
+        required: true
+        default: './slp/tests/acceptance/integration_test.staging.postman_environment.json'
+jobs: 
+  acceptance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: extenda/selfservice-lossprevention-common
+          path: 'slp'
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up secrets
+        uses: extenda/actions/gcp-secret-manager@v0
+        with:
+          service-account-key: ${{ inputs.secret-auth }}
+          secrets: |
+            GITHUB-TOKEN: github-token
+
+      - name: Set up GCloud
+        uses: extenda/actions/setup-gcloud@v0
+        id: gcloud
+        with:
+          service-account-key: ${{ inputs.gcloud-auth-staging }}
+
+      - name: Identity token
+        uses: extenda/actions/identity-token@v0
+        with:
+          service-account-key: ${{ inputs.gcloud-auth-staging }}
+          service-account: basketauditfrontend@${{ steps.gcloud.outputs.project-id }}.iam.gserviceaccount.com
```

Comment:
steps.gcloud.outputs.project-id why is this dynamic? Are we expecting diff env. 
If so then line 50 has inputs.gcloud-auth-**staging**. Wven that should be dynamic

Reply:
Just rename the variable to gcloud-auth